@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -132,13 +135,76 @@ export interface DataSourceTileList {
     /**
      * List of tiles we want to render but they might not be renderable yet (e.g. loading).
      */
-    visibleTiles: Tile[];
+    visibleTiles: Array<Tile | TileProxy>;
 
     /**
      * List of tiles that will be rendered. This includes tiles that are not in the visibleTiles
      * list but that are used as fallbacks b/c they are still in the cache.
      */
-    renderedTiles: Tile[];
+    renderedTiles: Array<Tile | TileProxy>;
+}
+
+/**
+ * Creates a unique key based on the supplied parameters. Note, the uniqueness
+ * is bounded by the bitshift. The mortonCode supports currently up to 26
+ * levels, the bitshift reduces this accordingly, so given the default bitshift
+ * of four, we support up to 22 levels. Given the current support up to level 19
+ * this should be fine.
+ * @param uniqueTileKey The unique tile key from which to compute the unique key.
```

Comment:
empty line before @param is standard, and indentation with two tabs for the following line looks nicer...

Reply:
Done, missed this one unfortunately